crates/sui-graphql-rpc/src/consistency.rs

```diff
@@ -43,16 +46,46 @@ pub(crate) struct ConsistentNamedCursor {
     pub c: u64,
 }
 
-/// The high checkpoint watermark stamped on each GraphQL request. This is used to ensure
-/// cross-query consistency.
-#[derive(Clone, Copy)]
-pub(crate) struct CheckpointViewedAt(pub u64);
+/// Watermark used by graphql queries to ensure cross-query consistency and flag epoch-boundary
```

Comment:
super tiny nit.

```suggestion
/// Watermark used by GraphQL queries to ensure cross-query consistency and flag epoch-boundary
```

Reply:
Also, given the suggestion to name `background_tasks` as `watermark_task` below, it might be worth moving this type there too?

Reply:
it is done